packages/code-du-travail-api/src/server/routes/search/search.savedResults.js

```diff
@@ -1,4 +1,4 @@
-const knownQueries = require("@cdt/data...datafiller/prequalified.json");
+const knownQueries = require("@cdt/data...datafiller/prequalified.data.json");
```

Comment:
good idea to prefix our json data with `.data.json`

Reply:
yep i just copied what you've done elsewhere :)